subiquity/server/controllers/shutdown.py

```diff
@@ -102,6 +102,10 @@ async def copy_logs_to_target(self, context):
         else:
             await arun_command(
                 ['cp', '-aT', '/var/log/installer', target_logs])
+            if os.path.exists("/var/log/cloud-init.log"):
+                # Preserve ephemeral boot cloud-init logs if applicable
+                await arun_command(
+                    ['cp', '-aT', '/var/log/cloud-init*.log', target_logs])
```

Comment:
The above copy is a directory, yours are files.  I think you want this, to avoid the error `cp: extra operand`:
```suggestion
                    ['cp', '-a', '/var/log/cloud-init*.log', target_logs])
```

Reply:
Thanks copy-pasta issue there. will test and confirm behavior on this PR .

Reply:
Note that this is a funny one to test, because the above copy is after the logs are copied to target, so the debug logs cut off abruptly before this `arun_command()`